File: Code/App/v2/RTG/Scripts/App/Models/RTGExaminationViewModel.js
function RTGExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.PatientInfo = new PatientsModel(data.PatientInfo);
    this.RTGComment = new
        RTGExaminationCommentViewModel(data.RTGComment);
    this.examinationDescriptionRTG = ko.observable();

};

File: Code/App/separateDB/RTG/ViewModels/RTGExaminationViewModel.cs
using RepositoryClasses.Models;

namespace RTG.ViewModels
{
    public class RTGExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public RTGExaminationCommentViewModel RTGComment { get; set; }
        public int PatientDieseaseId { get; set; }
    }
}